libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -538,7 +521,7 @@ static VerifiableCredential vcExperianFraudFailed() {
                         .build();
         return generateVerifiableCredential(
                 TEST_SUBJECT,
-                Cri.EXPERIAN_KBV,
+                EXPERIAN_FRAUD,
```

Comment:
Should I be worried that there were no broken tests with this the wrong way around?

Reply:
Maybe... I think it indicates how few parts of our code actually care about the VC type.

(It broke because the evidence validator now uses this explicitly, rather than trying to guess the VC type based on the evidence scores...)